stable_baselines/gail/dataset/record_expert.py

```diff
@@ -32,13 +32,14 @@ def generate_expert_traj(model, save_path, env=None, n_timesteps=0,
     # Retrieve the environment using the RL model
     if env is None and isinstance(model, BaseRLModel):
         env = model.get_env()
+        if env.num_envs > 1:
```

Comment:
You should do that check after the `if isinstance(env, VecEnv):` otherwise it will break with normal gym env.

Reply:
I put it under the wrong `if` *_*